Щубетов Артур ДЗ for Lesson 1

artourio opened the pull request:
Задание к первому уроку выполнено. Жду вашего комментария.

artourio committed:
the first task is done
create and print some vars

artourio committed:
task_2 is done. Comments inside the .py file

artourio committed:
task_3 done

artourio committed:
task_4 done

artourio committed:
task_5 done

artourio committed:
task_6(7) done

artourio committed:
Update task_3.py

artourio committed:
Update task_4.py

DmitryChitalov commented on Урок 1. Практическое задание/task_1.py:
```diff
@@ -11,3 +11,10 @@
+print('Ваши данные для входа в аккаунт: \n',
+      'Имя:', usr_name, '\n',
+      'Пароль:', usr_passwd, '\n',
+      'Возраст:', usr_old)
```
здесь желательно применить ф-строки

DmitryChitalov commented on Урок 1. Практическое задание/task_2.py:
```diff
@@ -9,3 +9,14 @@
+минут или секунд.
+"""
+print('Вы ввели', sec_amount, 'секунд. В формате чч : мм : сс это составит:')
+print("%.1f" % usr_hours, ':', "%.1f" % usr_minutes, ':', usr_sec)
```
также нужны ф-строки

artourio replied:
Извините, но не "нужны ф-строки", а "здесь возможно использовать ф-строки". Не вижу смысла забивать гвозди микроскопом

DmitryChitalov commented on Урок 1. Практическое задание/task_3.py:
```diff
@@ -8,3 +8,9 @@
+term_2 = int(usr_numb + usr_numb)
+term_3 = int(usr_numb + usr_numb + usr_numb)
+total = term_1 + term_2 + term_3
+print('Сумма i + ii + iii составляет:', total)
```
также ф-строки

DmitryChitalov commented on Урок 1. Практическое задание/task_4.py:
```diff
@@ -10,3 +10,14 @@
+        usr_numb //= 10
+        if digit >= mx_digit:
+            mx_digit = digit
+    print('Самая большая цифра в числе:', mx_digit)
```
выполнено

DmitryChitalov commented on Урок 1. Практическое задание/task_5.py:
```diff
@@ -18,3 +18,18 @@
+          "%.2f" % staff_profit)
+else:
+    print('Финансовый результат - убыток. Его величина:', abs(profit))
+    print('Казна пустеет, милорд. Подданные будут голодать')
```
а если равны?
здесь нужно if elif else

DmitryChitalov commented on Урок 1. Практическое задание/task_6.py:
```diff
@@ -19,3 +19,15 @@
+    day += 1
+    print(day, '-й', 'день: ', mid_res, ' км')
+print('Ответ: на ', day, '-й', 'день спортсмен достигнет результата - не менее ',
+      target_result, 'км.')
```
выполнено

artourio committed:
modified:   task_1.py - formating through f-string
	modified:   task_2.py - formating through f-string
	modified:   task_3.py - formating through f-string
	modified:   task_4.py - formating through f-string
	modified:   task_5.py - extend the solution if profit == 0 (zero)
	modified:   task_6.py - formating through f-string

artourio committed:
Merge branch 'lesson_1' of https://github.com/artourio/python_basics into lesson_1

artourio committed:
modified:   task_3.py
	modified:   task_4.py

artourio committed:
modified:   "\320\243\321\200\320\276\320\272 1. \320\237\321\200\320\260\320\272\321\202\320\270\321\207\320\265\321\201\320\272\320\276\320\265 \320\267\320\260\320\264\320\260\320\275\320\270\320\265/task_2.py"
	modified:   "\320\243\321\200\320\276\320\272 1. \320\237\321\200\320\260\320\272\321\202\320\270\321\207\320\265\321\201\320\272\320\276\320\265 \320\267\320\260\320\264\320\260\320\275\320\270\320\265/task_5.py"

DmitryChitalov commented on Урок 1. Практическое задание/task_1.py:
```diff
@@ -11,3 +11,10 @@
+print(f'Ваши данные для входа в аккаунт:\n'
+      f'Имя: {usr_name} \n'
+      f'Пароль: {usr_passwd} \n'
+      f'Возраст: {usr_old} \n')
```
выполнено

DmitryChitalov commented on Урок 1. Практическое задание/task_2.py:
```diff
@@ -9,3 +9,14 @@
+минут или секунд.
+"""
+print(f'Вы ввели {sec_amount} секунд. В формате чч : мм : сс это составит:')
+print(f'{usr_hours:.1f} : {usr_minutes:.1f} : {usr_sec:.1f}')
```
выполнено

DmitryChitalov commented on Урок 1. Практическое задание/task_3.py:
```diff
@@ -8,3 +8,9 @@
+term_2 = int(usr_numb + usr_numb)
+term_3 = int(usr_numb + usr_numb + usr_numb)
+total = term_1 + term_2 + term_3
+print(f'Сумма i + ii + iii составляет: {total}')
```
выполнено

DmitryChitalov commented on Урок 1. Практическое задание/task_4.py:
```diff
@@ -10,3 +10,14 @@
+        usr_numb //= 10
+        if digit >= mx_digit:
+            mx_digit = digit
+    print(f'Самая большая цифра в числе: {mx_digit}')
```
выполнено
